ios/gutenberg/GutenbergViewController.swift

```diff
@@ -185,6 +185,10 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
     func gutenbergDidRequestImagePreview(with fullSizeUrl: URL, thumbUrl: URL?) {
         print("Gutenberg requested fullscreen image preview for " + fullSizeUrl.absoluteString)
     }
+
+    func gutenbergDidRequestMediaEditor(with mediaUrl: URL, callback: @escaping MediaPickerDidPickMediaCallback) {
+        print("Gutenberg requested media editor for " + mediaUrl.absoluteString)
```

Comment:
It would be nice to test the callback too, just to be sure.

We can return a random media url (for example `https://cldup.com/_rSwtEeDGD.jpg` taken from https://frontenberg.tomjn.com). In this way, if the proper block is updated with the given URL, we know everything is working 👍

Reply:
Do you mean a UI Test? There's any similar test example you can point me to?

Reply:
This is a good example: https://github.com/wordpress-mobile/gutenberg-mobile/blob/40f4c19eaa2147cc8165332af68776f338471f92/ios/gutenberg/GutenbergViewController.swift#L71:L71

The idea would be, by selecting that option in the demo app, to see something happening.
On the real app we would edit the image and upload the edited version of it. Here in the demo app we can simulate a new edited version by giving back this new url. In this way we ensure that the method and callback work properly.

Does it make sense? Maybe I'm confused 😅

Reply:
Done in 2257b1c3d889555be7ade0a9418047ee7e40dd9a